x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/otel_logs/multi_integration_install_banner.tsx

```diff
@@ -22,7 +22,7 @@ export function MultiIntegrationInstallBanner() {
 
   const { performRequest, requestState } = useInstallIntegrations({
     onIntegrationCreationFailure,
-    packages: ['system', 'kubernetes'],
+    packages: ['system_otel', 'kubernetes_otel'],
```

Comment:
Just to double check as description doesn't mention it, is `kubernetes_otel` also a replacement for `kuernetes`?

Reply:
yes